src/app/main-page/main-page.html

```diff
@@ -0,0 +1,11 @@
+<div class="img">
+    <div class="left">
+        <logo></logo>
```

Comment:
all custom elements need a name with at least two words and a hyphen
https://stackoverflow.com/questions/22545621/do-custom-elements-require-a-dash-in-their-name